src/chime_dash/assets/sidebar.js

```diff
@@ -0,0 +1,40 @@
+const copy_to_clipboard = () => {
+    // Copies url to clipboard and changes the button text to let the user know it happened.
+    const el = document.createElement('textarea');
+    const button = document.getElementById('copy_params');
+    let original_text = button.textContent;
+    el.value = window.location.href;
+    document.body.appendChild(el);
+    el.select();
+    document.execCommand('copy');
+    document.body.removeChild(el);
+    button.textContent = "URL copied to clipboard";
+    let t = setTimeout(function(button, original_text){button.textContent = original_text;}, 2500, button, original_text);
+    return el.value;
+};
+
+const reset_parameters = () => {
```

Comment:
I don't think that we want to tie the reset functionality to the save to/load from URL feature. If we ever take that in a different direction, it would break the parameter reset. This approach also results in a full page reload even if no parameters are reset.